### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by MishaHnatyshyn**

### CODE REVIEW

**Comment by nechaido** [comment]
Задание?

**Comment by MishaHnatyshyn** [comment]
> Задание?

Variant 5. Level A.

Build a list, the elements of which are the words of the text. Words are separated by commas, after the last word is a point. Print all words of a given length.

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#include <string>
+#include <list>
+
+using namespace std;
+
+list<string> generateList(string str){
+    list<string> list;
+    char separator = ',';
+    int start = 0 , length = 0;
+    for (int i = 0; i <= str.length()-1; ++i) {
```
Comment (nechaido) [inline main.cpp]:
```suggestion
    for (int i = 0; i < str.length(); i++) {
```

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        if (str[i] == separator || i == str.length()-1){
+            length = i - start;
+            string temp = str.substr(start,length);
+            start +=length + 1;
```
Comment (nechaido) [inline main.cpp]:
```suggestion
            start = i + 1;
```

**Comment by MishaHnatyshyn** [comment]
@nechaido

**Review by nechaido — APPROVED** [review]
Принято

**Pull request merged**